service_planner/models/service_allocate.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api
+import datetime
+
+
+class ServiceAllocate(models.Model):
+    """
+    Allocated service with definition of all the template components
+    """
+
+    # model
+    _name = 'service.allocate'
+    _description = 'Allocate service'
+
+    # fields
+    # template service reference
+    service_template_id = fields.Many2one('service.template',
+                                          string='Template service',
+                                          required=True,
+                                          )
+    # container service reference
+    service_container_id = fields.Many2one('service.container',
+                                           string='Container service',
+                                           required=True,
+                                           )
+    # dedicated color
+    service_color = fields.Char('Color',
+                                related='service_template_id.base_color')
+
+    # assigned vehicles
+    vehicle_ids = fields.Many2many('fleet.vehicle', string='Vehicles')
+    # assigned employee
+    employee_ids = fields.Many2many('hr.employee', string='Team')
+    # employee names
+    employee_names = fields.Char('Employees', compute='_compute_emply_name', store=True)
+    # assigned equipment
+    equipment_ids = fields.Many2many('maintenance.equipment', string='Equipment')
+
+    # locality reference
+    locality = fields.Char('Locality')
+
+    # scheduled start time
+    start_sched = fields.Datetime('Start scheduled', required=True)
+    # scheduled start time
+    stop_sched = fields.Datetime('Stop scheduled',
```

Comment:
In generale, i nomi variabili li terrei per esteso (es. scheduled non ha più lettere di container).

Reply:
ok, rinominata